flink-python/dev/lint-python.sh

```diff
@@ -854,3 +863,8 @@ pushd "$FLINK_PYTHON_DIR" &> /dev/null
 if [ $skip_checks -eq 0 ]; then
     check_stage
 fi
+
+if [[ ${CLEAN_UP_FLAG} -eq 1 ]]; then
+    rm -rf ${CONDA_HOME}
+    rm -rf ${STAGE_FILE}
```

Comment:
Should we also clean up the `flink-python/.tox` folder?

Reply:
Yes. Good catch. Although the `.tox` directory is not generated during the `e2e` stage. The .tox folder is generated during the `flink-python` module test stage.

Reply:
It looks like tirggering the tox test in `e2e` stage. I need to correct the script.